.github/workflows/gha_generate_readme_and_issuetemp.yml

```diff
@@ -19,28 +19,27 @@ jobs:
       - name: Checkout repo
         uses: actions/checkout@v3
 
-      - name: Setup Conda
-        uses: s-weigand/setup-conda@v1
+      - name: Setup Micromamba ${{ matrix.python-version }}
+        uses: mamba-org/setup-micromamba@v1
         with:
-          activate-conda: false
-          conda-channels: conda-forge
-
-      - name: Build environment
-        shell: bash -l {0}
-        run: |
-          conda env create -f environment.yml
+          environment-file: environment.yml
+          init-shell: bash
 
       - name: Generate the README and ISSUE TEMP from JSON source
-        run: |
-          source activate cefi-info-hub
+        shell: bash -l {0}
+        run: >
           python generate_readme.py
-          python generate_issue_template.py
+          && python generate_issue_template.py
+
+      - name: Check Links
+        run:  python -m pytest --check-links
```

Comment:
We don't check the JSON directly but we check the generated md and all the other files that contains links for docs. That way we avoid checking the JSON twice.

Reply:
While that would be faster it is not much faster and we do have links in other places, no? I recommend to check all the links to avoid any rot in the docs and/or repo. If speed is a concern we can restrict it to just the generated md file.